.github/actions/helm-dryrun/action.yml

```diff
@@ -0,0 +1,45 @@
+name: "Run helm dry-run"
```

Comment:
@gionn wdyt about this action in alfresco-build-tools please? I took it from here: https://github.com/Activiti/activiti-cloud-full-chart/tree/master/.github/actions/helm-dryrun

Reply:
not a problem to move there but https://github.com/Activiti/activiti-cloud-full-chart/blob/master/.github/actions/helm-dryrun/action.yml#L30 looks something specific to your repos?

Reply:
I'm not familiar with the logic enough TBH. I'll duplicate this action in a third repo for now and will check this with Elias before merging, thanks

Reply:
I got what you were saying reworking the code :+1: 
I'll extract this the common logic to alfresco-build-tools, and will wait for Elias to merge.

Reply:
The action has been extracted (see https://github.com/Alfresco/alfresco-build-tools/pull/164) : will review Activiti workflows once that works on alfresco repositories